vnext/Shared/IWebSocketResource.h

```diff
@@ -86,7 +86,7 @@ struct IWebSocketResource {
   static std::shared_ptr<IWebSocketResource>
   Make(const std::string &url, bool legacyImplementation = false, bool acceptSelfSigned = false);
```

Comment:
>Make [](start = 2, length = 4)

What about this one as noexcept?

Reply:
Decided not to. We currently pass error info through the existing exception if resource creation fails.

May revisit later.